website/docs/guides/best-practices/how-we-style/1-how-we-style-our-dbt-models.md

```diff
@@ -0,0 +1,68 @@
+---
+title: How we style our dbt models
+id: 1-how-we-style-our-dbt-models
+---
+
+## Fields and model names
+
+- 👥 Models should be pluralized, e.g. `customers`, `orders`, `products`.
+- 🔑 Each model should have a primary key.
+- 🔑 The primary key of a model should be named `<object>_id`, e.g. `account_id` – this makes it easier to know what `id` is being referenced in downstream joined models.
+- 🔑 Keys should be string data types.
+- 🔑 Consistency is key! Use the same field names across models where possible, e.g. a key to the `customers` table should be named `customer_id` rather than `user_id` or 'id'.
```

Comment:
```suggestion
- 🔑 Consistency is key! Use the same field names across models where possible. For example, a key to the `customers` table should be named `customer_id` rather than `user_id` or 'id'.
```